devices/dresden_elektronik.js

```diff
@@ -31,4 +32,31 @@ module.exports = [
         extend: extend.light_onoff_brightness(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['FLS-A lp (1-10V)'],
+        model: 'BN-600078',
+        vendor: 'Dresden Elektronik',
+        description: 'Zigbee controller for 1-10V/PWM',
+        extend: extend.light_onoff_brightness({noConfigure: true}),
+        exposes: [e.light_brightness().withEndpoint('l1'), e.light_brightness().withEndpoint('l2'),
+            e.light_brightness().withEndpoint('l3'), e.light_brightness().withEndpoint('l4')],
+        endpoint: (device) => {
+            return {'l1': 11, 'l2': 12, 'l3': 13, 'l4': 14};
+        },
+        meta: {multiEndpoint: true, disableDefaultResponse: true},
+        configure: async (device, coordinatorEndpoint, logger) => {
+            await reporting.bind(device.getEndpoint(11), coordinatorEndpoint, ['genLevelCtrl', 'genOnOff']);
+            await reporting.bind(device.getEndpoint(12), coordinatorEndpoint, ['genLevelCtrl', 'genOnOff']);
```

Comment:
Does this device have any physical buttons on it? If not, then we don't add reporting (to limit the amount of messages on the network)

Reply:
It does not. Sorry, I copied that from an example without fully realizing what it would do. Fixed.

(Which actually addressed that the device wouldn't always show the current state properly on start. Thanks!)